content/en/Platform Deep Dive/Pentests/pentest-types.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Comprehensive Pentests"
+linkTitle: "Comprehensive Pentests"
+weight: 5
+description: >
+  Learn more about Comprehensive Pentests that we offer.
+---
+
+{{% pageinfo %}}
+We're working to introduce more pentest types soon. Stay tuned.
+{{% /pageinfo %}}
+
+You can launch Comprehensive Pentests on our [PtaaS](/getting-started/glossary/#pentest-as-a-service-ptaas) platform. Refer to the table below for details.
+
+| | Comprehensive Pentest |
+| --- | --- |
+| **Definition** | {{% comprehensive-pentest %}} |
+| **Minimum Number of Credits per Pentest** | 5 credits, thereafter in increments of 1 credit<br><li>{{% comprehensive-big-scope %}}</li> |
+| **Pentest Scope** | Broad area of an [asset](/getting-started/glossary/#asset)<!--<br><li>The pentest is focused on breadth</li>--> |
+| **Use Cases** | <li>Comprehensive [security audit](/getting-started/glossary/#security-audit)</li><li>[Compliance audit](/getting-started/glossary/#compliance-audit) testing based on the frameworks such as [SOC 2](https://us.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc2report.html), [ISO 27001](https://www.iso.org/isoiec-27001-information-security.html), [PCI-DSS](https://www.pcisecuritystandards.org/), [CREST](https://www.crest-approved.org/), or [HIPAA](https://www.hhs.gov/hipaa/index.html)</li><li>M&amp;A due diligence</li><li>Internal or third-party attestation request</li> |
+| **[Pentest Lead](/getting-started/glossary/#pentest-lead) Assigned** | Yes |
+| **Available [Pentest Reports](/platform-deep-dive/pentests/reports/)** | <li>Customer Letter</li><li>Attestation Letter</li><li>Attestation Report</li><li>Full Report</li><li>Full Report + Finding Details</li> |
+| **Report Target Audience** | External stakeholders |
+
+<!-- You can change the type of your pentest before we move it to the [Planned](/penteststates/) state. Select **Edit** on the pentest brief, and then select the type under **Pentest Type**. -->
+
+## Next Steps
+
+{{% getting-started-steps %}}
```

Comment:
@yogipetkar I added updates for Comprehensive Pentests as per our discussion.

Deploy preview: https://deploy-preview-165--cobalt-docs.netlify.app/platform-deep-dive/pentests/pentest-types/

Future link in Product Docs: `https://developer.cobalt.io/platform-deep-dive/pentests/pentest-types/`